htmresearch/frameworks/nlp/classification_model.py

```diff
@@ -416,3 +420,19 @@ def _inferDocumentDetailed(self, tokenList, sortResults=True):
 
     else:
       return categoryVotes, sampleIdList, distancetoSampleIds
+
+
+  @staticmethod
+  def _sortArray(array, seed=42):
```

Comment:
Since the seed is always fixed here, the tie breaking will not be random - it will be a fixed order each time. Is that what you want?